Libraries/LibWeb/Painting/BorderPainting.cpp

```diff
@@ -18,6 +18,8 @@ static constexpr double dark_light_absolute_value_difference = 1. / 3;
 
 static Color light_color_for_inset_and_outset(Color const& color)
 {
+    if (color.alpha() == 0)
+        return Color::Transparent;
     auto hsv = color.to_hsv();
     if (hsv.value >= dark_light_absolute_value_difference)
         return Color::from_hsv(hsv);
```

Comment:
Hm, isn't it an issue that we're ignoring `color.alpha()` here? Look at how `Gfx::Color::saturated_to()` deals with the HSV roundtrip for example. Same goes for `dark_color_for_inset_and_outset`